src/main/resources/scripts/xslt/transformToMods.xsl

```diff
@@ -3253,17 +3253,210 @@
-      contains(field[@name='Rolle']/value, 'Motiv')">
-        <xsl:call-template name="mods_subject_ophav_person" />
-      </xsl:if>
+      <xsl:call-template name="mods_subject_ophav_person" />
     </xsl:for-each>
-
   </xsl:template>
   <!-- END subject -->
 
   <!-- START subject ophav person -->
   <xsl:template name="mods_subject_ophav_person">
+    <!-- Person-tabel Recipient (person) -->
+    <xsl:if test="(contains(field[@name='Rolle']/value, 'modtager') or
+    contains(field[@name='Rolle']/value, 'supplerende modtager')) and
+    not(field[@name='Kooperation']/value)">
+      <xsl:element name="mods:subject">
+        <xsl:element name="mods:name">
+          <xsl:attribute name="altRepGroup">rcp1</xsl:attribute>
+          <xsl:attribute name="displayLabel">Recipient</xsl:attribute>
+          <xsl:attribute name="type">personal</xsl:attribute>
+          <xsl:call-template name="cumulus_get_lang_attribute" />
+          <xsl:element name="mods:role">
+            <xsl:element name="mods:roleTerm">
+              <xsl:attribute name="type">code</xsl:attribute>
+              <xsl:value-of select="'rcp'" />
+            </xsl:element>
+          </xsl:element>
+          <xsl:if test="field[@name='Efternavn']/value">
+            <xsl:element name="mods:namePart">
+              <xsl:attribute name="type">family</xsl:attribute>
+              <xsl:value-of select="field[@name='Efternavn']/value" />
+            </xsl:element>
+          </xsl:if>
+          <xsl:if test="field[@name='Fornavn']/value">
+            <xsl:element name="mods:namePart">
+              <xsl:attribute name="type">given</xsl:attribute>
+              <xsl:value-of select="field[@name='Fornavn']/value" />
+            </xsl:element>
+          </xsl:if>
+          <xsl:if test="field[@name='Født']/value or field[@name='Død']/value">
+            <xsl:element name="mods:namePart">
+              <xsl:attribute name="type">date</xsl:attribute>
+              <xsl:value-of select="field[@name='Født']/value" />
+              <xsl:value-of select="'/'" />
+              <xsl:value-of select="field[@name='Død']/value" />
+            </xsl:element>
+          </xsl:if>
+          <xsl:if test="field[@name='Nationalitet']/value">
+            <xsl:element name="mods:description">
+              <xsl:value-of select="field[@name='Nationalitet']/value" />
+            </xsl:element>
+          </xsl:if>
+          <xsl:if test="field[@name='Titel']/value">
+            <xsl:element name="mods:namePart">
+              <xsl:attribute name="type">termsOfAddress</xsl:attribute>
+              <xsl:value-of select="field[@name='Titel']/value" />
+            </xsl:element>
+          </xsl:if>
+          <xsl:if test="field[@name='Andet navn']/value">
+            <xsl:element name="mods:displayForm">
+              <xsl:value-of select="field[@name='Andet navn']/value" />
+            </xsl:element>
+          </xsl:if>
```

Comment:
Andet navn bør beholde den tidligere struktur med get_lang:
        <xsl:if test="field[@name='Andet navn']/value">
          <xsl:element name="mods:displayForm">
            <xsl:for-each select="field[@name='Andet navn']/value">
              <xsl:call-template name="cumulus_get_lang_attribute" />
              <xsl:call-template name="cumulus_get_value" />
            </xsl:for-each>
          </xsl:element>
        </xsl:if>